src/pages/Appointments/AppointmentsPage.tsx

```diff
@@ -107,19 +107,30 @@ function DateRangeDisplay({ dateFrom, dateTo }: DateRangeDisplayProps) {
 
   const today = new Date();
 
-  // Case 1: Today only
+  // Case 1: Today only and Yesterday only
```

Comment:
nitpick: i don't think a date value can be "today" AND "yesterday" together at the same time. It could be either "today" OR "yesterday".

```suggestion
  // Case 1: Today only or Yesterday only
```